docs/core/concepts/persistence.md

```diff
@@ -47,7 +46,7 @@ Note that when running Prefect Core locally, your Tasks' cached states will be s
 
 ## Checkpointing
 
-Oftentimes it is useful to persist your Task's data in an external location.  You could always write this logic directly into the Task itself, but this can sometimes make testing difficult.  Prefect offers a notion of Task "checkpointing" which ensures that every time the Task is successfully run, it's [result handler](results.html#result-handlers) is called.  To configure your Tasks for checkpointing, simply provide a result handler and set `checkpoint=True` at Task initialization:
+Oftentimes it is useful to persist your Task's data in an external location. You could always write this logic directly into the Task itself, but this can sometimes make testing difficult. Prefect offers a notion of Task "checkpointing" which ensures that every time the Task is successfully run, it's [result handler](results.html#result-handlers) is called. To configure your Tasks for checkpointing, provide a result handler and set `checkpoint=True` at Task initialization:
```

Comment:
```suggestion
Oftentimes it is useful to persist your task's data in an external location. You could always write this logic directly into the `Task` itself, but this can sometimes make testing difficult. Prefect offers a notion of task "checkpointing" that ensures that every time a task is successfully run, its [result handler](results.html#result-handlers) is called. To configure your tasks for checkpointing, provide a result handler and set `checkpoint=True` at task initialization:
```